redis/redis_test.go

```diff
@@ -34,10 +36,17 @@ func Test_Redis_Set_Override(t *testing.T) {
 
 	err = testStore.Set(key, val, 0)
 	require.NoError(t, err)
+
+	keys, err := testStore.Keys()
```

Comment:
i have seen that you always test the new function in other tests 
but would prefer if we had a separate function for the keys functionality so we don't always bloat all the others

Reply:
@ReneWerner87 To be able to test `Keys()` there needs to be data. Doing it this way not only tests the Keys() function but also validates that the keys from other actions.

I can write a separate test for `Keys()` but we should still check that Keys() returns 1 in the test for `set()` after creating a key. Or that it returns 0 after deleting the last key().